### PR TITLE
Bump cosmos-sdk rev to v0.40.0-rc6

### PR DESCRIPTION
**Pull request by tony-iqlusion**
This is the commit hash from this upstream PR which fixes the proto definitions so they compile again:

https://github.com/cosmos/cosmos-sdk/pull/8236

### CODE REVIEW

**Review by jkilpatr — APPROVED** [review]
I notice the 'tendermint' files are not currently being used in the repo should we bother to check in the rust files in that case? We are depending on tendermint-proto for those definitions.

Otherwise good and the extra generated files are not offensive, just not needed.

File: cosmos-sdk-proto/src/prost/tendermint.p2p.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (tony-iqlusion) [inline cosmos-sdk-proto/src/prost/tendermint.p2p.rs:1]: @jkilpatr when you say:

> I notice the 'tendermint' files are not currently being used in the repo should we bother to check in the rust files in that case?

Were you talking about this file, or something else?

Clearly this file is bogus and should probably be excluded.

Comment (jkilpatr) [reply cosmos-sdk-proto/src/prost/tendermint.p2p.rs:1]: it seems that all the files starting with tendermint.* are totally unused and not compiled.

**Comment by tony-iqlusion** [comment]
Looks like cosmos-sdk v0.40.0-rc6 is imminent: https://github.com/cosmos/cosmos-sdk/pull/8249

**Comment by tony-iqlusion** [comment]
`v0.40.0-rc6` is tagged! 🎉

Went ahead and bumped this PR. There were no changes to the generated protos.

**Pull request merged**